ajax/libs/unitegallery/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "unitegallery",
+  "filename": "js/unitegallery.min.js",
+  "description": "The Unite Gallery is multipurpose javascript gallery based on jquery library. It's built with a modular technique with a lot of accent of ease of use and customization",
+  "version": "1.7.14",
+  "license": "MIT",
+  "homepage": "http://unitegallery.net/",
+  "author": {
+    "name": "Max Valiano",
+    "url": "https://github.com/vvvmax"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/vvvmax/unitegallery/"
+  },
+  "keywords": [
+    "unitegallery",
+    "Gallery Plugin",
+    "jQuery",
+    "Responsive"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/vvvmax/unitegallery.git",
+    "basePath": "package/unitegallery",
+    "files": [
+      "**/*.!(txt)"
+    ]
```

Comment:
Maybe you can discuss with its author and see if it's possible add a dist folder for release files?
I think this is similar to my case #7604.